BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -36,6 +36,65 @@ LoadComponentCallback (
   }
 }
 
+/**
+  Extend EPayload to TPM.
+
+  @param[in]  ContainerSig    Containetr Signature
+  @param[in]  ComponentName   Component Name
+
+  @retval RETURN_SUCCESS      Operation completed successfully.
+  @retval RETURN_NOT_FOUND    EPayload hash Not found
+**/
+RETURN_STATUS
+ExtendEpayload (

```

Comment:
Let us move it into Stage2Support.c file to make Stage2 main flow clean.

Reply:
sure